oauth2client/contrib/django_util/__init__.py

```diff
@@ -276,7 +382,7 @@ def __init__(self, request, scopes=None, return_url=None):
 
         # make sure previously requested custom scopes are maintained
         # in future authorizations
-        credentials = storage.get_storage(self.request).get()
+        credentials = self.credentials
```

Comment:
1. Don't call one part of the public API of an object from within the implementation of another part of the public API of the same object.
2. Don't call instance methods (including properties) from within the instance constructor.